workloads/upgrade-perf/common.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+set -x
+
+# Check cluster's health
+if [[ ${CERBERUS_URL} ]]; then
```

Comment:
We might want to add a Cerberus check at the end of the run as well to make sure the upgraded cluster is healthy. We can query ripsaw as well, for example: https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/network-perf/run_pod_network_test_fromgit.sh#L77. Thoughts?

Reply:
Think the benchmark will fail if Cerberus reports issues right? In that case we don't need an explicit check.